source/reference/access-mdb-shell-help.txt

```diff
@@ -0,0 +1,242 @@
+.. _mdb-shell-help:
+
+===========================
+Access the |mdb-shell| Help
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+``mongosh`` provides additional help information in its help system.
+This document provides an overview of accessing this help
+information.
+
+.. _mdb-shell-help-command-line:
+
+Command Line Help
+-----------------
+
+To see the list of options and help for starting ``mongosh``, use the
+``--help`` option from the command line:
+
+.. code-block:: sh
+
+   mongosh --help
+
+Shell Help
+----------
+
+.. include:: /includes/admonitions/help-interchangeable.rst
+
+To see a basic list of commands you can run, in ``mongosh``, type
+``help``:
+
+.. code-block:: javascript
+
+   help
+
+.. _mdb-shell-help-db:
+
+Database Help
+-------------
+
+In ``monogsh``:
+
+- To see the list of databases on the server, use the ``show dbs``
+  command:
+
+  .. code-block:: javascript
+
+     show dbs
+
+  ``show databases`` is an alias for ``show dbs``.
+
+- To see the list of help for methods you can use on the ``db``
+  object, call the ``db.help()`` method or run the ``db.help`` command:
+
+  .. include:: /includes/admonitions/help-interchangeable.rst
```

Comment:
I thought we were getting rid of these includes in favor of the top-level callout, no?

Reply:
my bad. I removed one and commented on github, but forgot to go back and remove the others. 🥇 

now it's done.